From f252fd6db430b1617083c57371fb722dbc7f82a6 Mon Sep 17 00:00:00 2001
From: ran shalit <rshalit@habana.ai>
Date: Thu, 20 May 2021 21:38:55 +0300
Subject: [PATCH] [SW-42811] gaudi: pcie: increase timeout to link

Increase timeout on waiting for link up, as we found
scenarios in which the timeout from device power up
till hw is ready in host side is longer than 5 seconds
and can reach to ~15seconds. A timeout of 60 seconds
was decided as a time interval which is good enough
for most scenarios, yet allowing to continue in preboot
flow, so that it is serving other interfaces too such
as BMC.

Change-Id: I65da145a4b1ecf0f89286cd071a268ed3cd5f39d
Signed-off-by: ran shalit <rshalit@habana.ai>
---

diff --git a/gaudi/common.h b/gaudi/common.h
index adbef9a..4b79513 100644
--- a/gaudi/common.h
+++ b/gaudi/common.h
@@ -47,7 +47,7 @@
 #define GAUDI_DMA_CHANNELS_NUM		8
 #define GAUDI_QM_CGM_NUM		2
 
-#define PCIE_WAIT_TIMEOUT_US		(5 * USEC_PER_SEC)
+#define PCIE_WAIT_TIMEOUT_US		(60 * USEC_PER_SEC)
 #define BTL_SYNC_TIMEOUT_US		(USEC_PER_SEC / 2)
 #define PCIE_EXT_LD_TIMEOUT_US		576
 
